src/editors/containers/EditorContainer/index.test.jsx

```diff
@@ -36,16 +37,23 @@ describe('EditorContainer component', () => {
       beforeEach(() => {
         el = shallow(<EditorContainer {...props}>{testContent}</EditorContainer>);
       });
-      test('close behavior is linked to modal onClose and footer onCancel', () => {
+      test('close behavior is linked to modal onClose', () => {
```

Comment:
Header and footer need to be mocked for the snapshots.
and you should query by finding by the element type, not by children index (unless doing both)